app.rb

```diff
@@ -62,7 +75,7 @@ def auth_server_uri
     aal = params[:aal] || '2'
     skip_encryption = params[:skip_encryption]
     request_url = request.create(saml_settings(ial: ial, aal: aal))
-    request_url += "&skip_encryption=#{skip_encryption}" if skip_encryption
+    request_url += { skip_encryption: skip_encryption }.to_query if skip_encryption
```

Comment:
Do we need to keep the `&` ? It's not generated by `to_query`

```suggestion
    request_url += '&' + { skip_encryption: skip_encryption }.to_query if skip_encryption
```

(Not sure if we have Rubocop configured to get angry about string concatenation like this)

Reply:
Great catch, will fix